tox.ini

```diff
@@ -88,9 +88,9 @@ commands =
 description = check code style, e.g. with flake8
 skip_install = true
 deps =
-    flake8
+    pyproject-flake8
 commands =
-    flake8 {posargs}
+    pflake8 {posargs}
```

Comment:
`flake8` does not currently support reading from `pyproject.toml`, so the wrapper `pflake8` does the same. Perhaps this configuration should be in a `.flake8` file instead.

Reply:
I've since removed usage of `pflake8` in favor of keeping `flake8` configuration in `setup.cfg`